src/glmtools.jl

```diff
@@ -499,6 +499,19 @@ false
 dispersion_parameter(D) = true
 dispersion_parameter(::Union{Bernoulli, Binomial, Poisson}) = false
 
+"""
+    _safer_int(x::T)
+    
+Convert to Int, when `x` is within 1 eps of an integer.
+"""
+function _safer_int(x::T) where {T<:Base.IEEEFloat}
```

Comment:
Why not allow any `AbstractFloat`?

Reply:
The logic here is based on IEEE 754 behavior though I guess it's a reasonable bit of logic for finite precision type with `eps()` defined. And `eps()` is defined on `AbstractFloat`, so okay :smile

Reply:
```suggestion
function _safer_int(x::T) where {T<:AbstractFloat}
```